jamovi/surv.r.yaml

```diff
@@ -18,24 +18,44 @@ items:
           type: text
           content: ($key)
           visible: (groups)
-        - name: censored
-          title: Censored
-          type: integer
```

Comment:
i could see a count of censored events being useful in some contexts ...

Reply:
I was in doubt as to keep it or not but I think you are right. It's probably better to include it as well. I made the change.